src/caja-desktop-window.c

```diff
@@ -291,9 +380,12 @@ realize (GtkWidget *widget)
     details = window->details;
 
     /* Make sure we get keyboard events */
-    gtk_widget_set_events (widget, gtk_widget_get_events (widget)
+    GdkDisplay *display = gdk_screen_get_display (gdk_screen_get_default());
+    if (GDK_IS_X11_DISPLAY (display))
+        gtk_widget_set_events (widget, gtk_widget_get_events (widget)
                            | GDK_KEY_PRESS_MASK | GDK_KEY_RELEASE_MASK);
```

Comment:
add 4 spaces to align the start of the line with the opening parenthesis:
```suggestion
        gtk_widget_set_events (widget, gtk_widget_get_events (widget)
                               | GDK_KEY_PRESS_MASK | GDK_KEY_RELEASE_MASK);
```

Reply:
Found and fixed